demos/classification_demo/python/classification_demo.py

```diff
@@ -37,73 +37,94 @@
+
     io_args.add_argument('--loop', default=False, action='store_true',
-                         help='Optional. Enable reading the input in a loop.')
-    io_args.add_argument('-o', '--output', required=False,
-                         help='Optional. Name of the output file(s) to save.')
-    io_args.add_argument('-limit', '--output_limit', required=False, default=1000, type=int,
-                         help='Optional. Number of frames to store in output. '
-                              'If 0 is set, all frames are stored.')
-    io_args.add_argument('--no_show', help="Optional. Don't show output.", action='store_true')
-    io_args.add_argument('--output_resolution', default=None, type=resolution,
-                         help='Optional. Specify the maximum output window resolution '
-                              'in (width x height) format. Example: 1280x720. '
-                              'Input frame size used by default.')
-    io_args.add_argument('-u', '--utilization_monitors', default='', type=str,
-                         help='Optional. List of monitors to show initially.')
+        help='enable reading the input in a loop')
+
+    io_args.add_argument('-noshow', '--noshow', action='store_true',
+        help="don't show output")
+
+    io_args.add_argument('-o', '--output', required=False, metavar="<OUTPUT>",
+        help='name of the output file(s) to save')
+
+    io_args.add_argument('-res', '--res', default=None, type=resolution, metavar="<STRING>",
+        help='set image grid resolution in format WxH')
+
+    io_args.add_argument('-u', '--utilization_monitors', default='', type=str, metavar="<MONITORS>",
+        help='resource utilization graphs. '
+            'c - average CPU load, d - load distribution over cores, m - memory usage, h - hide')
 
     input_transform_args = parser.add_argument_group('Input transform options')
+
     input_transform_args.add_argument('--reverse_input_channels', default=False, action='store_true',
-                                      help='Optional. Switch the input channels order from '
-                                           'BGR to RGB.')
-    input_transform_args.add_argument('--mean_values', default=None, type=float, nargs=3,
-                                      help='Optional. Normalize input by subtracting the mean '
-                                           'values per channel. Example: 255.0 255.0 255.0')
-    input_transform_args.add_argument('--scale_values', default=None, type=float, nargs=3,
-                                      help='Optional. Divide input by scale values per channel. '
-                                           'Division is applied after mean values subtraction. '
-                                           'Example: 255.0 255.0 255.0')
+        help='switch the input channels order from BGR to RGB')
+
+    input_transform_args.add_argument('--mean_values', default=None, type=float, nargs=3, metavar="<STRING>",
```

Comment:
Without metavar specification, the `-h` produces the following message:

```
--mean_values MEAN_VALUES MEAN_VALUES MEAN_VALUES
                        normalize input by subtracting the mean values per
                        channel. Example: 255.0 255.0 255.0
```
which is aligned with README. However, with `metavar="<STRING>"` we obtain
```
 --mean_values <STRING> <STRING> <STRING>
```

Firstly, with the metavars introduction the README should be updated as well for each argument. 
Next, is there any reason to create for some args specific metavars, like `"<DEVICE>"`, `"<OUTPUT>"`, while for other args common `"<STRING>"`, `"<NUMBER>"`?

Reply:
In https://github.com/openvinotoolkit/open_model_zoo/pull/3172 @Wovchena said that every non-bool variable should have a meta var. But I guess I just picked the wrong names for them. What are your suggestions for these names?